contracts/Controller.sol

```diff
@@ -310,7 +322,30 @@ contract Controller is ReentrancyGuard, Ownable {
      * @notice deposit long option into vault
      * @param _args DepositArgs structure
      */
-    // function _depositLong(Actions.DepositArgs memory _args) internal {}
+    function _depositLong(Actions.DepositArgs memory _args) internal returns (MarginAccount.Vault memory) {
+        require(_args.from == msg.sender, "Controller: depositor address and msg.sender address mismatch");
```

Comment:
wasn't the conclusion for args.from also be acceptable to be the vault owner?
Let's say msg.sender is an operator, the operator can also specify the `owner` to be the args.from. In that case our marginPool will just try to pull assets from the vault owner.

Reply:
for later down the line, not right now.

Reply:
not in this version, we said its too dangerous to open that up.